Composer installation broken on PHP 5.5

Looks like the composer setup script fail to fetch the signature file, probably because the image certificates are not recent enough.
```
All settings correct for using Composer
Downloading...
The "https://getcomposer.org/download/2.2.18/composer.phar.sig" file could not be downloaded: SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed
Failed to enable crypto
failed to open stream: operation failed
Retrying...
The "https://getcomposer.org/download/2.2.18/composer.phar.sig" file could not be downloaded: SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed
Failed to enable crypto
failed to open stream: operation failed
Retrying...
The "https://getcomposer.org/download/2.2.18/composer.phar.sig" file could not be downloaded: SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed
Failed to enable crypto
failed to open stream: operation failed
The download failed repeatedly, aborting.
CMake Error: The source directory "/var/www/html" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
```

--- a/5.5/Dockerfile
+++ b/5.5/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -ksSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/5.6/Dockerfile
+++ b/5.6/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/7.0/Dockerfile
+++ b/7.0/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/7.1/Dockerfile
+++ b/7.1/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/7.2/Dockerfile
+++ b/7.2/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/7.3/Dockerfile
+++ b/7.3/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/7.4/Dockerfile
+++ b/7.4/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/8.0/Dockerfile
+++ b/8.0/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/8.1/Dockerfile
+++ b/8.1/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/8.2-rc/Dockerfile
+++ b/8.2-rc/Dockerfile
@@ -60,7 +60,13 @@ RUN \
     apk add --no-cache --virtual build-deps ${BUILD_DEPS} && \
     \
     # Install PHP extensions
-    install-php-extensions @composer-${COMPOSER_VERSION} ${PHP_EXTS} && \
+    install-php-extensions ${PHP_EXTS} && \
+    \
+    # Install composer
+    curl -sSL https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \
+    chmod +x /usr/bin/composer && \
+    echo -ne "\033[01mComposer installed\033[00m: " && \
+    composer --version && \
     \
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

--- a/update.sh
+++ b/update.sh
@@ -11,6 +11,8 @@ generate_dockerfile(){
 
     image="${version}-fpm-alpine"
 
+    [ "${version}" = "5.5" ] && curl_flags='-ksSL' || curl_flags='-sSL'
+
     dockerfile=./${version}/Dockerfile
     cat > ${dockerfile} <<TEMPLATE
 #
@@ -75,7 +77,13 @@ RUN \\
     apk add --no-cache --virtual build-deps \${BUILD_DEPS} && \\
     \\
     # Install PHP extensions
-    install-php-extensions @composer-\${COMPOSER_VERSION} \${PHP_EXTS} && \\
+    install-php-extensions \${PHP_EXTS} && \\
+    \\
+    # Install composer
+    curl ${curl_flags} https://getcomposer.org/download/\${COMPOSER_VERSION}/composer.phar -o /usr/bin/composer && \\
+    chmod +x /usr/bin/composer && \\
+    echo -ne "\\033[01mComposer installed\\033[00m: " && \\
+    composer --version && \\
     \\
     # Install support for locales
     # @see https://github.com/gliderlabs/docker-alpine/issues/144

